src/Cardano/Wallet/Fee.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE RankNTypes #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- Provides the API of Fee calculation.
+
+
+module Cardano.Wallet.Fee
```

Comment:
No need for an extra module here, let's have that in `Cardano.Wallet.CoinSelection` :+1: